rubocop/.rubocop.yml

```diff
@@ -0,0 +1,10 @@
+require:
+  - rubocop-performance
+  - rubocop-rails
+  - rubocop-rspec
+
+inherit_from:
+  - ./.rubocop-common.yml
+  - ./.rubocop-performance.yml
+  - ./.rubocop-rails.yml
+  - ./.rubocop-rspec.yml
```

Comment:
The most common use case is to just copy down this file and you are all set. If you want to pick and choose the rubocop plugins, you can write your own `.rubocop.yml` and inherit just the stuff you need.